index.html

```diff
@@ -431,12 +449,71 @@
+with a time value equal to the time immediately before the user agent starts
+the <a href="http://www.w3.org/TR/html5/browsers.html#unloading-documents">unload event</a> of the previous document.
+
+If there is no previous document or the previous document has a
+different <a href="http://tools.ietf.org/html/rfc6454">origin</a> than the
+<a>current document</a>, this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
+with a time value equal to zero.</p>
+</dd>
+
+<dt>readonly attribute DOMHighResTimeStamp unloadEventEnd</dt>
+<dd>
+<p>If the previous document and the <a>current document</a> have the
+same <a href="http://tools.ietf.org/html/rfc6454">origin</a>,
+this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
+with a time value equal to the time immediately after the user agent finishes the
+<a href="http://www.w3.org/TR/html5/browsers.html#unloading-documents">unload event</a> of the previous document. If there is no previous document or the previous document has
+a different <a href="http://tools.ietf.org/html/rfc6454">origin</a> than the
+<a>current document</a> or the unload is not yet completed, this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
+with a time value equal to zero.</p>
+
+<p>If there are HTTP redirects or
+<a href="http://www.w3.org/TR/html5/infrastructure.html#concept-http-equivalent-codes" title='HTTP response codes equivalence'>equivalent</a>
+when navigating and not all the redirects or equivalent are from the
+same <a href="http://tools.ietf.org/html/rfc6454">origin</a>,
+both <a href="#widl-PerformanceNavigationTiming-unloadEventStart">unloadEventStart</a> and
+<a href="#widl-PerformanceNavigationTiming-unloadEventEnd">unloadEventEnd</a> MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
+with a time value equal to zero.</p>
+</dd>
+
 <dt>readonly attribute DOMHighResTimeStamp domLoading</dt>
 <dd>
 <p>This attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
 with a time value equal to the time immediately before the user agent sets the
 <a href="http://www.w3.org/TR/html5/dom.html#current-document-readiness">current
-document readiness</a> to
+document readiness</a> of the <a>current document</a> to
 <a href="http://www.w3.org/TR/html5/dom.html#current-document-readiness" title='"loading" document readiness'>"loading"</a> [[!HTML5]].</p>
   <p class="warning">The <code>domLoading</code> attribute is <strong>deprecated</strong> and may be removed in future versions of this specification. Due to differences in when a Document object is created in existing user agents, the value returned by the domLoading is implementation specific and should not be used in meaningful metrics.</p>

```

Comment:
Good point. We could the following:
- drop domLoading from PerformanceNavigationTiming
- move the deprecation note to PerformanceTiming.domLoading, in the historical section.